### DISCUSSION TITLE
How can I remove ">" from the internal links?

### DISCUSSION THREAD

**Discussion by carlos-truong** [ObsidianPublisher/obsidian-github-publisher#310] [Q&A — Plugin] [answered]
Hi,

if I add a internal link to a heading, for example "[[#Heading 1]]", it is converted to "[[#Heading 1| > Heading 1]]" after uploading to the GitHub repository by Obsidian Publisher.
I don't know why. How can I remove ">" from the internal links?

**Comment by Mara-Li** [comment]
Sorry, it's the default Obsidian behavior. You can set a regex to remove it.

**Comment by carlos-truong** [comment]
Hi Lisandra,
Thank you for your reply!

Is it really the behavior of Obsidian? Because it changes when uploading to the repo so I assumed it was caused by Obsidian Publisher. Is it right?

**Reply by Mara-Li** [reply]
It's complicated, tbh. It's just a way to set it. You can use regex to remove it.

**Comment by carlos-truong** [comment]
Could you guide me how to remove it by Regex?
Tbh, I don't know how to use Regex!

**Reply by Mara-Li** [reply, marked as answer]
`/\[{2}#(.*)\| > (.*)\]{2}/gi` → `[[$1|$2]]`

**Reply by carlos-truong** [reply]
Could I ask where to put it? Is this right?
<img width="419" alt="image" src="https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/29748838/afc658a4-9c87-4f82-8665-da5ff1e9b2ae">

**Reply by Mara-Li** [reply]
Yes

**Reply by carlos-truong** [reply]
It doesn't work.
- On my vault: [[#Heading 1]]
- On the repo, it's still: [[#Heading 1| > Heading 1]]

**Reply by Mara-Li** [reply]
Set the arrow to down to use the regex after the other transformation.

**Reply by carlos-truong** [reply]
It works now! Thank you so much :D
